File: programming files/even_odd_site.py
// Open HTML file 
with open("numbers.html", "w") as f:
    
    // Write to HTML file
    f.write("<html>\n<head>\n<title>List of Numbers</title>\n</head>\n<body>\n")
    f.write("<table>\n<tr><th>Even Numbers</th><th>Odd Numbers</th></tr>\n")
    
    // Even-Odd calc and write to HTML
    for i in range(1, 50):
        if i % 2 == 0:
            f.write("<tr><td>{}</td><td></td></tr>\n".format(i))
        else:
            f.write("<tr><td></td><td>{}</td></tr>\n".format(i))
    f.write("</table>\n</body>\n</html>")

// Write HTML file to console
with open("numbers.html", "r") as f:
    print(f.read())
    

File: programming files/even_odd_site.java


import java.io.*;

public class even_odd_site {
    public static void main(String[] args) throws IOException {
        
        // Create a BufferedWriter Object
        BufferedWriter file = new BufferedWriter(new FileWriter("numbers.html"));
        
        // Write to a HTML file
        file.write("<html>\n<head>\n<title>List of Numbers</title>\n</head>\n<body>\n");
        file.write("<table>\n<tr><th>Even Numbers</th><th>Odd Numbers</th></tr>\n");
        
        // Even-Odd calc and output to HTML
        for (int i = 1; i <= 50; i++) {
            if (i % 2 == 0) {
                file.write("<tr><td>" + i + "</td><td></td></tr>\n");
            }
            else {
                file.write("<tr><td></td><td>" + i + "</td></tr>\n");
            }
        }
        file.write("</table>\n</body>\n</html>");
        file.close();
        
        // New Buffered Reader Object
        BufferedReader reader = new BufferedReader(new FileReader("numbers.html"));
        
        // Print HTML file to console
        String line;
        while ((line = reader.readLine()) != null) {
            System.out.println(line);
        }
        reader.close();
    }
}
